CHANGELOG.md

```diff
@@ -36,6 +36,10 @@ our [guidelines for writing a good changelog entry](https://github.com/biomejs/b
 
 ### JavaScript APIs
 
+#### Enhancements
+
+- Add new lint rule, `useTopLevelRegex`, which enforces defining regular expressions at the top level of a module. Contributed by @dyc3.
+
```

Comment:
This should go in the `### Linter` section

Reply:
In a `### New features` subsection of `## linter`